src/app/[locale]/(pages)/esci/conferma/page.tsx

```diff
@@ -7,13 +7,22 @@ import { WebProfileApi } from '@/api/webProfileApiClient';
 import NoSessionActiveComp from '@/app/[locale]/_component/noSessionActiveComp/noSessionActiveComp';
 import { storageUserOps } from '@/app/[locale]/_utils/storage';
 import { SessionState } from '@/api/generated/webProfile/SessionState';
+import { trackEvent } from '@/app/[locale]/_utils/mixpanel';
 
 const LogoutConfirm = (): React.ReactElement => {
   const [sessionData, setSessionData] = useState<SessionState>();
   const t = useTranslations('ioesco');
   const userFromStorage = storageUserOps.read();
   const isL1 = userFromStorage?.spidLevel === process.env.NEXT_PUBLIC_JWT_SPID_LEVEL_VALUE_L1;
 
+  useEffect(() => {
+    if (sessionData) {
+      trackEvent(isL1 ? 'IO_SESSION_EXIT_STATUS_PAGE' : 'IO_PROFILE_SESSION_EXIT_STATUS_PAGE', {
+        session_status: sessionData.session_info.active ? 'on' : 'off',
```

Comment:
This has been use already twice. Would we consider to put this login inside an utility function?

Reply:
Where else has it been used?

Reply:
[Here](https://github.com/pagopa/io-web-profile/blob/494f73370586a7cdecc36f2b4366bbdf67f5a6c2/src/app/%5Blocale%5D/page.tsx#L29) if I'm not wrong. Check it out...

Reply:
you're right, i will create common utility